presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1086,7 +1114,7 @@ core::PlanNodePtr VeloxQueryPlanConverter::toVeloxQueryPlan(
           node->source)) {
     std::optional<core::JoinType> joinType = std::nullopt;
     if (equal(node->predicate, semiJoin->semiJoinOutput)) {
-      joinType = core::JoinType::kLeftSemi;
+      joinType = core::JoinType::kLeftSemiFilter;
```

Comment:
Is this change related here?

Reply:
This is to pick up Masha's change, per ask